### PR TITLE
HSH-210: Update howto for the code setup and SampleQC

### PR DESCRIPTION
**Pull request by GennadiyZakharov**

### CODE REVIEW

File: docs/wes-qc-hail.md [hunk 1]
```diff
@@ -44,6 +67,27 @@ export PYTHONPATH=/path/to/wes-qc
...
+In the `scripts` folder you can find two scripts.
+* `hlrun_local` - runs the Python script via `spark-submit`. You need to run it on the spark master node on your cluster.
+* `hlrun_remote` - runs the code on the Spark cluster form your local machine.
+It performs a series of operations:
+  * Sync the codebase to the remote cluster, defined by the
+  * Create tmux session on the remoter cluster
+  * Run the Python script via `hlrun_local`
+  * Attach to the tmux session to monitor the progress
+
+The second script is use
```
Comment (Eric-Kobayashi) [inline docs/wes-qc-hail.md:81]: another broken sentence here

File: docs/wes-qc-hail.md [hunk 2]
```diff
@@ -52,6 +96,17 @@ Load VCFs into Hail and save as a Hail MatrixTable
...
+This resource set is required for the super-population prediction on the population PCA step.
+
+```shell
+spark-submit 1-import_data/1-import_gatk_vcfs_to_hail.py --all
+```
+
+You need to create this resource set only once.
+Then you can reuse it with any data cohort.
+
+
```
Comment (Eric-Kobayashi) [inline docs/wes-qc-hail.md:109]: need to add docs for step 1/4

File: docs/wes-qc-hail.md [hunk 3]
```diff
@@ -145,19 +207,19 @@ Next an input table is generated to run the random forest on.
 spark-submit 3-variant_qc/2-create_rf_ht.py
 ```
 
-Next train the random forest model, note the use of --master local[*] here which is needed to ensure that the worker nodes pick up the correct python modules.
+Next, train the random forest model, note the use of --master local[*] here which is needed to ensure that the worker nodes pick up the correct python modules.
```
Comment (Eric-Kobayashi) [inline docs/wes-qc-hail.md:210]: where is the --master local[*] arg?

File: docs/wes-qc-hail.md [hunk 4]
```diff
@@ -44,6 +67,27 @@ export PYTHONPATH=/path/to/wes-qc
 >spark-submit /path/to/hail_script.py
 >```
 
+### Automatically sync and run the code from a local machine
+
+In the `scripts` folder you can find two scripts.
+* `hlrun_local` - runs the Python script via `spark-submit`. You need to run it on the spark master node on your cluster.
+* `hlrun_remote` - runs the code on the Spark cluster form your local machine.
+It performs a series of operations:
+  * Sync the codebase to the remote cluster, defined by the
```
[the remark this answers is not in the document]
Comment (GennadiyZakharov) [reply docs/wes-qc-hail.md:76]: Thanks. Fixed.

File: docs/wes-qc-hail.md (hunk 1 above)
Comment (GennadiyZakharov) [reply docs/wes-qc-hail.md:81]: Fixed

File: docs/wes-qc-hail.md (hunk 2 above)
Comment (GennadiyZakharov) [reply docs/wes-qc-hail.md:109]: These are the docs for step 1/4. I incorrectly pasted the script's name. It's fixed now.

File: docs/wes-qc-hail.md (hunk 3 above)
Comment (GennadiyZakharov) [reply docs/wes-qc-hail.md:210]: In the latest Hail, the RF model training works correctly in a parallel environment. I removed this sentence and placed it in the note below.

**Review by Eric-Kobayashi — APPROVED** [review]

**Pull request merged**